ci/datadog-agent-values-dd.yaml

```diff
@@ -0,0 +1,18 @@
+agents:
+  image:
+    tag: 7.51.0-rc.1
+    tagSuffix: jmx
+datadog:
+  apiKey: $DD_API_KEY
+  apm:
```

Comment:
If you use UDS to connect to the trace-agent, then this is not needed, as the UDS socket is on by default when deploying agent via helm: https://docs.datadoghq.com/containers/kubernetes/apm/?tab=unixdomainsocketuds#configure-the-datadog-agent-to-collect-traces.

UDS is the recommended approach, and contains some benefits for things like container tagging. This [PR](https://github.com/DataDog/opentelemetry-demo/pull/25) shows how to get it working with UDS, where on the client you mount the socket path.

That said, this isn't a blocker and will still work/ is fine as is.

Reply:
That's good to know, I'd like to leave the TCP as-is while looking into UDS later. Previously I wasn't able to ge the OTel APIs + DD SDKs to work without TCP.